### DISCUSSION TITLE
Build your own interface

### DISCUSSION THREAD

**Discussion by SebastianS92** [agronick/aa-torque#151] [Q&A]
I saw you can start and create your own design which is more than I could ever hoped for when looking for a solution like this. Just asking: is there any way to customize the entire layout and if yes, which files do I need to tinker with?

**Comment by SebastianS92** [comment]
So I found out the visual setup is done app/src/main/res/layout/fragment_dashboard which then includes all needed files like fragment_gauge.

I would like a vertical or horizontal bar and setting up the XML wouldn't be much of a problem. Hoewever that's only the vidual side. I think the actual computing side is designed for round instruments only?
Also only doing that setup won't bring me much further as the app itself won't know how to set it up

**Comment by agronick** [comment]
See this guide. https://github.com/agronick/aa-torque/pull/63
